src/doc-builders.js

```diff
@@ -2,7 +2,7 @@
 
 function assertDoc(val) {
   /* istanbul ignore if */
-  if (process.env.NODE_ENV === "production") {
+  if (process.env.NODE_ENV !== "production") {
```

Comment:
Not sure this is the right way of doing it, a lot of people will run prettier in `"development"` env.

Reply:
@azz I updated the PR, can you check again?

Reply:
To clarify, when building the bundle, we will replace that check with `"production"` to remove that entirely

Reply:
Oh, right. Cool.